rostime/include/ros/time.h

```diff
@@ -256,8 +256,41 @@ namespace ros
     static bool isSystemTime() { return true; }
   };
 
+  /**
+   * \brief Time representation.  Always monotonic-clock time.
```

Comment:
It should be noted here that it is unaffected by ROS time, just like "wall" time is not.